WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -597,56 +573,25 @@ extension SiteStatsInsightsTableViewController: NoResultsViewControllerDelegate
             refreshInsights()
         }
 
-        if asyncLoadingActivated {
-            hideNoResults()
-            return
-        }
-
-        updateNoResults(title: NoResultConstants.loadingTitle,
-                        accessoryView: NoResultsViewController.loadingAccessoryView()) { noResults in
-                            noResults.hideImageView(false)
-        }
+        hideNoResults()
```

Comment:
We can remove the `defer` above and add `addViewModelListeners()` and `refreshInsights()` after `hideNoResults()`